docs/tr/docs/python-types.md

```diff
@@ -144,58 +144,95 @@ Yalnızca `str` değil, tüm standart Python tiplerinin bildirebilirsiniz.
 {!../../../docs_src/python_types/tutorial005.py!}
 ```
 
-### Tip parametreleri ile Generic tipler
+### Tip Parametreleri ile Genel Tipler
 
-"dict", "list", "set" ve "tuple" gibi diğer değerleri içerebilen bazı veri yapıları vardır. Ve dahili değerlerinin de tip belirtecleri olabilir.
+`dict`, `list`, `set` ve `tuple` gibi başka değerler içerebilen bazı veri yapıları vardır. Ve dahili değerler de kendi türlerine sahip olabilirler.
 
-Bu tipleri ve dahili tpileri bildirmek için standart Python modülünü "typing" kullanabilirsiniz.
+Dahili tipleri olan bu tiplere "**generic**" (jenerik) tipler denir. Ve bunları iç tipleriyle bile belirtmek mümkündür.
 
-Bu tür tip belirteçlerini desteklemek için özel olarak mevcuttur.
+Bu tipleri ve dahili tpileri bildirmek için standart Python küyüphanesi `typing`i kullanabilirsiniz. Bu tür tip belirteçlerini desteklemek için özel olarak mevcuttur.
```

Comment:
Bu tipleri ve dahili tipileri belirtmek için standart Python modülü olan `typing` modülünü kullanabilirsiniz. Modülün amacı tip belirteçlerini desteklemektir.